mxcube3/ui/components/Tasks/Characterisation.js

```diff
@@ -1,137 +1,293 @@
+'use strict';
+
 import React from 'react'
 import {reduxForm} from 'redux-form';
-import { Modal } from "react-bootstrap";
+import Modal from 'react-modal';
```

Comment:
What is "react-modal" ? How does it compare with the normal Modal window that comes with React ?
Normally I use "react-bootstrap" because it is the Bootstrap implementation for React, it is pretty convenient.
If react-modal doesn't bring any added value let's remove this dependency ?